### ISSUE TITLE
about the PIPA dataset

### ISSUE THREAD

**Issue by Verherder** [IFBigData/GRIT#2]
Hello, I am preparing for my graduate.
I tried to find the PIPA dataset in the Internet, since the pulic link to the dataset has gone. But nothing was found.

Could you share the PIPA? Thanks very much in advance.
Looking for your reply.

**Comment by cteant** [comment]
Hi, you can download the dataset from the following link:
link：https://pan.baidu.com/s/1JgTEJw6VTgVIidJt5FKt9w
password：pmv5

**Comment by Verherder** [comment]
> Hi, you can download the dataset from the following link: link：https://pan.baidu.com/s/1JgTEJw6VTgVIidJt5FKt9w password：pmv5

Get it.
Sincerely Thanks!

**Comment by Verherder** [comment]
> Hi, you can download the dataset from the following link: link：https://pan.baidu.com/s/1JgTEJw6VTgVIidJt5FKt9w password：pmv5

Sorry to borer you again. 
I am aiming at people recogniton.
But I found the data in baidu-disk is PIPA-relation dataset, which only obtains part of the orginal PIPA dataset.

Do you have the orginal PIPA dataset? (images are ok.)

**Comment by cteant** [comment]
> 

Sorry, we don't have the original PIPA dataset.

**Comment by Verherder** [comment]
Anyhow, thanks for your reply.

**Comment by Verherder** [comment]
Hi, 
I am trying to make the datatset work. But the annotation is confusing.

For example, the annotation looks like:
"72157641996575204 12987546783 508 124 376 1075 1 1 f1_72157641996575204_12987546783.jpg male young_adult", 
The first two are photo ids, the following the "508 124 376 1075" seems to be a bounding box. But after drawing it, the box is not a head box or a body box.

What does the "508 124 376 1075" mean?

**Comment by cteant** [comment]
> 

Yes, it is a bounding box, but its format is (x, y, w, h), the last two numbers represent the width and height of the bounding box. Try to draw the bounding box using (x, y, x+w, x+h).

**Comment by Verherder** [comment]
Yes, it is in format of (x, y, w, h), but the bbox looks like 
![image](https://user-images.githubusercontent.com/69044198/202171997-9ca8b9a4-fbee-4c43-9eec-d3942543942e.png)

It is confusing.

**Comment by jjb123456** [comment]
> Yes, it is in format of (x, y, w, h), but the bbox looks like ![image](https://user-images.githubusercontent.com/69044198/202171997-9ca8b9a4-fbee-4c43-9eec-d3942543942e.png)
> 
> It is confusing.

Hello, I am currently in the process of finding the pipa dataset for my research, but I am facing the same issue as you. I would like to know if you have found the pipa dataset by now. If you have, could you please share it? Alternatively, have you found any alternative datasets that can be used in place of the pipa dataset? If you have, could you please share them?

**Comment by NKSagarReddy** [comment]
Hi @cteant @grainrainn ,
If you don't mind, could you share the PIPA-relation dataset again, the previous link to the dataset seems to be expired.

I have tried PIPA-dataset crawling with FlickrAPI, but around 20% of the links to the images are expired, I would like to recover the PIPA-relation dataset..

**Comment by Verherder** [comment]
@NKSagarReddy  you can download it from: https://pan.baidu.com/s/1LDfzTTvyGtIJnqZHoxv55A  with the password: qw5c

**Comment by imjohnzakkam** [comment]
@grainrainn Could you please share the dataset link again? 

I think it is expired, I have viewed the link today, maybe the expiration is set for 30 days.

**Comment by Verherder** [comment]
> @grainrainn Could you please share the dataset link again?
> 
> I think it is expired, I have viewed the link today, maybe the expiration is set for 30 days.

链接: https://pan.baidu.com/s/1Tw2fzVxQhxfjwOfUFA7-Tg 提取码: ktw6

**Comment by UUUserna** [comment]
@grainrainn Could you please share the dataset link again?
Thanks a lot！

**Comment by Verherder** [comment]
> [@grainrainn](https://github.com/grainrainn) Could you please share the dataset link again? Thanks a lot！

you can try with https://pan.baidu.com/s/1Og-sOSYFBSSvpkJi1UNLRQ?pwd=6rft
